unittest/test_whole_body_state.py

```diff
@@ -267,6 +267,36 @@ def test_communication_with_non_locked_joints(self):
                 S[1], _S[1], "Wrong contact friction coefficient at " + name
             )
 
+    def test_update_model(self):
+        model = pinocchio.buildSampleModelHumanoid()
+        pub = WholeBodyStateRosPublisher(model, "whole_body_state_update_model")
+        sub = WholeBodyStateRosPublisher(model, "whole_body_state_update_model")
+        time.sleep(1)
+        # publish whole-body state messages
+        names = model.names.tolist()
+        new_parameters = []
+        for name in names:
+            psi = pinocchio.Inertia().Random().toDynamicParameters()
```

Comment:
As we are just testing that the communication works, I don't think it matters

Reply:
We are converting the base position and velocity to whole-body COM position and velocities. Could you double-check that this is OK to do with not fully physically consistent parameters?

I'm OK to keep the code as it is if this is not needed. But, we need to be sure as this can be hard to track later.

Reply:
I included checks for the whole-body states after modifying the inertial parameters.